eng/pipelines/MIDebugEngine-CI.yml

```diff
@@ -1,5 +1,15 @@
 ---
 name: $(Date:yyyMMdd).$(Rev:r)
+
+schedules:
+# Run on the 15th and 30th of every month
```

Comment:
>30th [](http://example.com/codeflow?start=22&length=4)

Is it an issue that there will not be a run between February 15th and March 15th?

Reply:
I thought about this last night. I'll switch it to 1st and 15th to be even across the year
Although the max length of time we have is 30 days.

Reply:
and I did change that in the cron, just not the description :D